src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1264,6 +1266,13 @@ services:
         tags:
             - { name: form.type }
 
+    form.type.sell.product.seo_type:
```

Comment:
This file is now 1200 lines long.
Should we start putting product forms in a dedicated YAML file (and maybe later we'll create more YAML files to split this file, regrouped by domain?)

Reply:
yep, I suggested the split months (if not years) ago So I'm all for it but in a dedicated PR
I would even perform the full split by domain instead of just for product But this will create lots of conflicts we must be careful with pending PRs